src/style/panel/panel.less

```diff
@@ -193,6 +193,7 @@
     .button-reset-default();
     min-height: 56px;
     align-items: center;
+    padding: 1rem;
```

Comment:
added padding to align with design
Elements that exceed the min-height have no padding and look squished
<img width="261" height="177" alt="image" src="https://github.com/user-attachments/assets/0d19d6dd-65b8-4b5e-aff6-bf7b82b69444" />